tests/cli/test_correct_scraper.py

```diff
@@ -15,8 +15,8 @@
 
 tests = {
     'release': {
-        'args': ['-v', '23.0.1'],
-        'fname': 'firefox-23.0.1.en-US.linux64.tar.bz2',
+        'args': ['-v', '23.0.1', '-p', 'win32'],
```

Comment:
Nice I will remember. First we have to switch to pytest which will be done via another issue.